swift_package/BrainFlow/Sources/BrainFlowBindings/MLModule.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  MLModule.swift
+//  a Swift binding for BrainFlow's MLModule API
+//
+//  Created by Scott Miller on 9/22/21 on behalf of Aeris Rising, LLC.
+//
+
+import Foundation
+
+struct BrainFlowModelParams: Encodable {
+    let metric: BrainFlowMetrics
+    let classifier: BrainFlowClassifiers
+    let file: String
+    let other_info: String
+    let output_name: String
+    let max_array_size = 8192
+
+    init (metric: BrainFlowMetrics, classifier: BrainFlowClassifiers,
+          file: String = "", otherInfo: String = "", outputName: String = "") {
+        self.metric = metric
+        self.classifier = classifier
+        self.file = file
+        self.other_info = otherInfo
+        self.output_name = outputName
+    }
+}
+
+struct MLModule {
+    //MLModule class used to calc derivative metrics from raw data
+    
+    let modelParams: BrainFlowModelParams
+    
+    static func setLogLevel(_ logLevel: LogLevels) throws {
+        //set BrainFlow log level, use it only if you want to write your own messages to BrainFlow logger,
```

Comment:
is it how you specify docstrings in swift? In other files i saw java style docstrings with:
/*
 *
 */
It should be unified among files

Reply:
I found the following guide on the Apple Developer website:

https://developer.apple.com/documentation/xcode/writing-symbol-documentation-in-your-source-files

Will that work?

Reply:
Main request is to keep it uniform among files, ideally to make docstrings compatible with doxygen, this way it will be possible to publish it to brainflow docs

Reply:
The Internet says doxygen does not work for Swift. Apple recommends DocC instead. It is built into Xcode. I used it to generate this sample:

<img width="1504" alt="image" src="https://github.com/brainflow-dev/brainflow/assets/73297684/af249c37-c552-456f-9dc6-3c5a0b328451">